components/config-mgmt-service/backend/elastic/suggestions.go

```diff
@@ -100,6 +187,9 @@ func (es Backend) getArrayAggSuggestions(term string, text string, filters map[s
 	}
 	// multiplying the size by 10 as elasticsearch will sort array aggregations by doc_count. Will trim it back to size once we match it again in go
 	aggs := elastic.NewTermsAggregation().Field(term).Size(SuggestionQuerySize)
+	if len(text) >= 2 {
+		aggs = aggs.Include(".*" + text + ".*")
+	}
```

Comment:
If the search box has two or more characters typed, then filter the aggregation with that value. Before when the aggregation was not filtered, the first 100 cookbooks would be used and if there were over 100 cookbooks then the last cookbooks would never be returned.